### PR TITLE
Make Replay installation instructions explicit in Playwright docs

### PR DESCRIPTION
**Pull request by Andarist**

### CODE REVIEW

**Review by jasonLaster — APPROVED** [review]
This is so much better.

File: src/app/test-runners/playwright/github-actions/page.md [hunk 1]
```diff
@@ -39,7 +36,10 @@ Now that the playwright config is set up, you can run the tests with `npx playwr
 
 ```yml {% fileName=".github/workflows/e2e.yml" lineNumbers=true %}
 name: End-to-end tests
-on: [push, pull_request]
+on:
+  pull_request:
+  push:
+    branches: [main]
```
Comment (bvaughn) [inline src/app/test-runners/playwright/github-actions/page.md:42]: Nice

File: src/app/test-runners/playwright/node.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ You can write tests as a function that uses `playwright.[browser].launch()`. Thi
 In this configuration, you’ll use the `getExecutablePath()` function from `@replayio/playwright` to ensure the Replay-enabled browser is used.
 
 <Callout type="info" emoji="👉">
- Passing the `RECORD_ALL_CONTENT: 1`  is only required for Replay Firefox.
+ Passing the `RECORD_ALL_CONTENT: 1`  is only required for Replay Chromium.
```
Comment (bvaughn) [inline src/app/test-runners/playwright/node.md:19]: Is this needed anymore?

File: src/app/test-runners/playwright/record-your-first-replay/page.md [hunk 3]
```diff
@@ -84,29 +80,26 @@ set REPLAY_API_KEY=<your_api_key>
...
       upload: true,
     }),
-    ['line'],
+    ["line"],
   ],
   projects: [
     {
-      name: 'replay-chromium',
-      use: { ...replayDevices['Replay Chromium'] },
+      name: "replay-chromium",
```
Comment (bvaughn) [inline src/app/test-runners/playwright/record-your-first-replay/page.md:97]: This naming pattern may cause screenshot snapshot tests to fail. We should talk about this.

**Review by bvaughn — APPROVED** [review]
We might have some dead files in here I think.

**Pull request merged**